spec/config_spec.rb

```diff
@@ -55,6 +55,15 @@
     end
   end
 
+  describe '#sidekiq_parallel' do
+    subject { patches_config.sidekiq_parallel }
+    before { patches_config.sidekiq_parallel = true }
+
+    it 'is the configured value' do
+      expect(subject).to eq(true)
+    end
```

Comment:
should we also be validating the result if config is not set?

Reply:
done